packages/manager/src/pages/profile/settings/profile-image/profile-image.validators.js

```diff
@@ -0,0 +1,22 @@
+import * as Yup from "yup";
+
+const profileImageValidationSchema = ({ t }) => {
```

Comment:
Remove the differences between investor and manager versions and move to `shared/components`